addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -90,18 +96,20 @@ export class Chatbot extends Record {
             return;
         }
         if (this.steps.at(-1)?.eq(this.currentStep)) {
-            const nextStep = await rpc("/chatbot/step/trigger", {
+            const storeData = await rpc("/chatbot/step/trigger", {
                 channel_id: this.thread.id,
                 chatbot_script_id: this.script.id,
             });
-            if (!nextStep) {
+            if (!storeData) {
                 this.currentStep.isLast = true;
                 return;
             }
-            this.steps.push(nextStep);
+            const { ChatbotStep: steps } = this.store.insert(storeData, { html: true });
+            this.steps.push(steps[0]);
```

Comment:
Ok to do later, but ideally chatbot record  could receive steps ADD from server and avoid js processing no?